pkgs/development/interpreters/python/hooks/python-imports-check-hook.sh

```diff
@@ -6,8 +6,17 @@ pythonImportsCheckPhase () {
 
     if [ -n "$pythonImportsCheck" ]; then
         echo "Check whether the following modules can be imported: $pythonImportsCheck"
-        export PYTHONPATH="$out/@pythonSitePackages@:$PYTHONPATH"
-        ( cd $out && eval "@pythonCheckInterpreter@ -c 'import os; import importlib; list(map(lambda mod: importlib.import_module(mod), os.environ[\"pythonImportsCheck\"].split()))'" )
+        pythonImportsCheckOutput=$out
+        if [ -n "$py" ]; then
+            echo "Using python specific output \$py for imports check"
+            pythonImportsCheckOutput=$py
+        fi
+        if [ -n "$python" ]; then
```

Comment:
This should be an `elif` instead, so we don't echo twice in case `$py` and `$python` are defined.